.github/workflows/ci.yml

```diff
@@ -118,6 +125,7 @@ jobs:
           # We use raw because our release process is not done on tag push (yet)
           tags: |
             type=raw,value=${{ steps.info.outputs.version }}
+            type=raw,value=${{ steps.info.version.latestTag }},enable=${{ !contains(steps.info.outputs.version, 'SNAPSHOT') }}
```

Comment:
I think the condition should test for a `-SNAPSHOT` suffix by using the `endsWith` command

Reply:
nice catch, omw to implement it 💪